### ISSUE TITLE
feature/update: red notification box

### ISSUE THREAD

**Issue by fedfomin** [openmultiplayer/web#518]
![unknown](https://user-images.githubusercontent.com/15944464/147772791-bfece884-4d3c-4b4d-beed-2e158fb81ec8.png)

While surfing the new forums i just noticed one of the (perhaps) main visual issues that i'd like to report, while surfing and testing out some features without being logged in, is that the red alert box (or notification box i suppose) while popping off gives some problems while reading a thread or doing something.

I was thinking it could be perhaps better to place it else where like on top right or top left.

**Comment by Southclaws** [comment]
Yeah this is a good point, it should be in the corner on desktop as to not cover content 

Also, out of interest how did you trigger these errors, were you intentionally doing it or did 8 errors just suddenly appear?

**Comment by fedfomin** [comment]
Yeah i replicated the error manually by reacting on emojis under the post without being logged in and did it a few times to trigger 8 errors.

**Comment by AmyrAhmady** [comment]
Irrelevant in new web.
